src/AppInstallerCLI.sln

```diff
@@ -105,6 +108,21 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		CodeAnalysis.ruleset = CodeAnalysis.ruleset
 	EndProjectSection
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AppInstallerCLIInterfaceServer", "AppInstallerCLIInterfaceServer\AppInstallerCLIInterfaceServer.vcxproj", "{2B00D362-AC92-41F3-A8D2-5B1599BDCA01}"

```

Comment:
Maybe consider removing CLI from the names since this is a generic interface that is not specific to CLI?

Reply:
In addition, we are trying to use `WinGet` rather than `AppInstaller` for new things.

Reply:
Made this change.